pkgs/development/libraries/mdds/default.nix

```
@@ -15,6 +17,7 @@ stdenv.mkDerivation rec {
   '';
 
   buildInputs = lib.optionals stdenv.cc.isClang [ llvmPackages.openmp ];
+  nativeBuildInputs = [ autoreconfHook ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook ];

  buildInputs = lib.optionals stdenv.cc.isClang [ llvmPackages.openmp ];

  postInstall = ''
    mkdir -p "$out/lib/pkgconfig"
    cp "$out/share/pkgconfig/"* "$out/lib/pkgconfig"
  '';
```

lets fix the ordering while we are at it